doc/datasetManager.md

```diff
@@ -0,0 +1,247 @@
+
+The configuration contains two parts:
+
+- the **graph indicators** represent macro indicators of the dataset content, displayed as small cards in the dataset
+  overview
+- the **categories**, displayed below the graph indicators, are used to provide more detailed insights for the different
+  types of elements inside the dataset
+
+### Graph indicators
+
+Configuration for the graph indicators must be defined with the key
+`[workspace].webApp.options.datasetManager.graphIndicators`.
+The value for this field must be an **array of objects**, where each object represents one indicator, with the
+following fields:
+
+- `id`: a unique identifier string for this indicator
+- `name`: an object with translations of the indicator name
+
+<details>
+<summary>JSON example</summary>
+
+```json
+{
+  "webApp": {
+    "options": {
+      "datasetManager": {
+        "graphIndicators": [
+          { "id": "entities", "name": { "en": "Entities", "fr": "Entités" } },
+          { "id": "relationships", "name": { "en": "Relationships", "fr": "Relations" } }
+        ]
+      }
+    }
+  }
+}
+```
+
+</details>
+
+<details>
+<summary>YAML example</summary>
+
+```yaml
+webApp:
+  options:
+    datasetManager:
+      graphIndicators:
+        - id: entities
+          name:
+            en: Entities
+            fr: Entités
+        - id: relationships
+          name:
+            en: Relationships
+            fr: Relations
+```
+
+</details>
+
+### Categories
+
+Configuration for the categories must be defined with the key
+`[workspace].webApp.options.datasetManager.categories`.
+The value for this field must be an **array of objects**, where each object represents one category, with the
+following fields:
+
+- `id`: a unique identifier string for this category
+- `name`: an object with translations of the category name
+- `type` _(optional)_: a string with one of the vales `"entity"` or `"relationship"` (leave to `null` to hide this
+  information)
+- `description` _(optional)_: an object with translations of the category description
+- `kpis` _(optional)_: an array of objects to represent key indicators of the category; each KPI object is defined with
+  the fields:
+  - `id`: a unique identifier string for this indicator
+  - `name`: an object with translations of the indicator name
+- `attributes` _(optional)_: an array of string values, to display the attributes of the category
+
+<details>
+<summary>JSON example</summary>
+
+```json
+{
+  "webApp": {
+    "options": {
+      "datasetManager": {
+        "categories": [
+          {
+            "id": "stock",
+            "name": { "en": "Stock", "fr": "Stock" },
+            "type": "relationship",
+            "description": {
+              "en": "A stock is a supply chain model entity representing location of part between operations.\nA stock contains one and only one part reference",
+              "fr": "Le stock est l'entité du modèle Supply Chain répresentant les biens entre les opérations."
```

Comment:
typo: r**E**présentant (without accent aigü, the same on line 154) :slightly_smiling_face: